docs/root/intro/arch_overview/upstream/service_discovery.rst

```diff
@@ -39,6 +39,13 @@ This means that care should be taken if active health checking is used with DNS
 to the same IPs: if an IP is repeated many times between DNS names it might cause undue load on the
 upstream host.
 
+If :ref:`respect_dns_ttl <envoy_api_field_Cluster.respect_dns_ttl>` respect_dns_ttl is enabled, 
```

Comment:
Take a look at the rendered docs, you will see that you are repeating the field names. Same elsewhere.